.github/workflows/nix.yaml

```
@@ -114,25 +111,44 @@ jobs:
+          curl -L "https://github.com/google/go-containerregistry/releases/download/v0.15.2/go-containerregistry_Linux_x86_64.tar.gz" | tar -xz -C $HOME/.local/bin crane
+          chmod +x $HOME/.local/bin/crane
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+      - name: Push Docker image with crane
+        if: steps.check-conditions.outputs.should_build == 'true' && (matrix.system == 'x86_64-linux' || matrix.system == 'aarch64-linux') && needs.check-conditions.outputs.should-push-docker == 'true'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Direct gunzip
+          gunzip -c result > image.tar
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)